pkg/scheduler/extender/extender.go

```diff
@@ -253,15 +255,20 @@ func (e *Extender) gatherCapacityRequestsByProvisioner(ctx context.Context, pod
 				return nil, baseerr.ErrorNotFound
 			}
 
-			ll.Debugf("PVC details: %s", pvc.String())
+			ll.Debugf("PVC %s status: %+v", pvc.Name, pvc.Status)
```

Comment:
Can we have just one log statement here if all of them have similar severity level?

Reply:
BTW shouldn't it be INFO?

Reply:
fixed

Reply:
I think it's too much for info
```
Nov 24 16:06:03.271 [DEBU] [Extender] [gatherCapacityRequestsByProvisioner] [web-1] [c75ace5c-24c6-42cc-81cc-7e60412aeee6] PVC www-web-1 status: {Phase:Pending AccessModes:[] Capacity:map[] Conditions:[]} spec: {AccessModes:[ReadWriteOnce] Selector:nil Resources:{Limits:map[] Requests:map[storage:{i:{value:62914560 scale:0} d:{Dec:<nil>} s:60Mi Format:BinarySI}]} VolumeName: StorageClassName:0xc000434750 VolumeMode:0xc000434760 DataSource:nil} SC: csi-baremetal-sc
```
@rkiselenko? @yurkov-anton?

Reply:
Do we really need to print all PVC details or only those which we are checking?

Reply:
I reduce log massage to print only spec and status. We can have invalid PVC (for example: accessMode or Limits) and it's better to store it here, if PVC is deleted or changed